apps/opik-backend/src/main/java/com/comet/opik/api/filter/ExperimentsComparisonField.java

```diff
@@ -7,10 +7,11 @@
 @Getter
 public enum ExperimentsComparisonField implements Field {
 
-    INPUT(INPUT_QUERY_PARAM, FieldType.STRING),
-    EXPECTED_OUTPUT(EXPECTED_OUTPUT_QUERY_PARAM, FieldType.STRING),
+    INPUT(NEW_INPUT_QUERY_PARAM, FieldType.STRING),
+    EXPECTED_OUTPUT(NEW_EXPECTED_OUTPUT_QUERY_PARAM, FieldType.STRING),
     OUTPUT(OUTPUT_QUERY_PARAM, FieldType.STRING),
-    METADATA(METADATA_QUERY_PARAM, FieldType.DICTIONARY),
+    METADATA(NEW_METADATA_QUERY_PARAM, FieldType.DICTIONARY),
```

Comment:
Important: this will work only be backwards compatible if the previous migration has backfilled all the data physically a the DB level. This is acceptable, but we have to clearly communicate it, as it can be confusing from the current description of the PR. 

The reason is that filters work actually at the DB level, so your manual backwards compatibly at the application level doesn't apply. The data has to be available in the actual table field.

Reply:
I was wondering about it. Either we can do something like if `input` (or some of the deprecated fields) then `(input = ? or data['input'])` or since the migration would guarantee it, we would have all data there. Also, the first PR ensures the new data persists in the data field as well. 

For this reason I thought this solution would be cleaner

Reply:
are we sure we need backwards compatibility?  what would happen if we make this change and disregard existing projects?

Reply:
Hi @amirzatcomet, I am not sure which projects you are referring to, but if projects are Opik users, those enum properties are only used internally. The only part that would be part of some public-facing API would be the enum name. Anyway, this implementation has changed since friday.

Reply:
@amirzatcomet we discussed that backwards compatibility isn't required. This comment was to highlight that fact, but the only downside would be the search not returning some dataset items. Anyway, the only requirement for this to work fine is just this previous PR to run properly for all records where it applies:
- https://github.com/comet-ml/opik/pull/397